l10n_br_fiscal/tools.py

```diff
@@ -89,3 +89,47 @@ def remove_non_ascii_characters(value):
         )
 
     return result
+
+
+def inform_journal_in_fiscal_operation(cr, company, values):
+    """
+    Create or Inform ir.property for link Journal with Fiscal Operation
```

Comment:
it's better to use fill_journal... or set_journal... than inform_journal... Fill/Set carry the notion of writing something when Inform would rather mean "reporting" about something.

Reply:
thanks @rvalyi for review, changed for **set_journal_...**